apps/web/README.md

```diff
@@ -12,16 +12,16 @@
 
 ### The desktop app?
 
-You can find all the desktop related code in [the `desktop/` directory](./desktop/). Since it uses the web app directly, we are keeping both together. (We should probably move it to it's own project at some point.)
+You can find all the desktop related code in the [desktop](./desktop/) directory. Since it uses the web app directly, we are keeping both together. (We should probably move it to it's own project at some point.)
```

Comment:
Let's keep the formatting as before here. It should look a directory `desktop/` instead of desktop.

Reply:
Ok